Snakefile

```diff
@@ -291,18 +291,28 @@ rule ml_analysis:
         hac_clusters_vertical = SEP.join([out_dir, '{dataset}-ml', 'hac-clusters-vertical.txt']),
         hac_image_horizontal = SEP.join([out_dir, '{dataset}-ml', 'hac-horizontal.png']),
         hac_clusters_horizontal = SEP.join([out_dir, '{dataset}-ml', 'hac-clusters-horizontal.txt']),
-        ensemble_network_file = SEP.join([out_dir,'{dataset}-ml', 'ensemble-pathway.txt'])
```

Comment:
Can we keep the ability to create a single ensemble pathway out of all pathways from all algorithms. It looks like there is a Snakemake rule for that, but we no longer tell Snakemake to create the output file here.

Reply:
I separated that into its own snakemake rule. This is because ensemble won't be run if any of the functions above it fail.

Reply:
I was mistaken and thought this deleted line was line 95 where the final input list is constructed.